php.twig/licenseinfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/php/twig/resources/TwigTemplate.html.twig</file>
+        <file>src/org/netbeans/modules/php/twig/resources/example.html.twig</file>
```

Comment:
@matthiasblaesing I added example files to `comment type="TEMPLATE_MINIMAL_IP"`. Should we move them to another comment type?

Reply:
There is a special type for that: `GUI_USABILITY` (the list is in: `nbbuild/antsrc/org/netbeans/nbbuild/extlibs/licenseinfo/CommentType.java`).

Reply:
I see. Thanks for the information!